config/providers.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+use Yiisoft\Yii\Filesystem\FileStorageServiceProvider;
+
+return [
+    'FileStorage' => FileStorageServiceProvider::class
```

Comment:
I think we should use an interface instead of the alias `FileStorage`

Reply:
It is not for a container